src/shardMigrationCheck.ts

```diff
@@ -0,0 +1,33 @@
+import {OldAndNewNodeResponse} from './utils/handlerResponses';
+import {getClusterHealth, getDocuments} from './elasticsearch/elasticsearch';
+import {ElasticsearchClusterStatus, Documents} from './elasticsearch/types';
+
+export async function handler(event: OldAndNewNodeResponse): Promise<OldAndNewNodeResponse> {
```

Comment:
We don't use information about the newest node in this lambda, but we need to pass the state through for the final step (removeNode)